src/public/assets/data/people.json

```diff
@@ -54,11 +60,11 @@
     "image": "",
     "bio": "Sam started with Red Dust in February 2013 after completing a Bachelor of General Studies at Monash University majoring in International Marketing in 1999. Sam’s previous role before Red Dust was with Treasury Wine Estate (Formerly Fosters Group); where she worked in Brand Marketing on several premium wine brands, including Penfolds, Seppelt and Coldstream Hills, for over five years."
   }, {
-    "name": "Jonathan Hermawan Tjapaltjarri",
+    "name": "Jonathan Lindsay-Tjapaltjarri Hermawan",
     "role": "Manager Central Australia",
     "category": "staff",
     "image": "",
-    "bio": "Jonathan is a fluent Pintupi-Luritja speaking man from the NT community of Walungurru (Kintore). He is a fully qualified teacher/educator and proud advocate for Aboriginal community leadership, youth empowerment and reconciliation. Based in Alice Springs, Jonathan currently works for Red Dust as an Alcohol and Men’s Health Educator and Coordinator, journeying with Aboriginal males (youth and adults) and their families."
+    "bio": "Jonathan is the Manager for Central Australia and also facilitates Red Dust's men and boys behaviour change and cultural leadership programs. He is multilingual and has cultural connections extending from Walungurru qualifications in teaching/education, alcohol and other drugs (AOD), automotive mechanics, music and theology. He is a strident advocate for Aboriginal community leadership, youth empowerment, many-ways learning and Reconciliation in action. "
```

Comment:
Good spot, the bio you removed had a curly quote in it. Did you do a global search for all curly single and double quotes in entire `app/` folder?